plugins/services/src/js/structs/Framework.js

```diff
@@ -1,15 +1,54 @@
 import { cleanServiceJSON } from "#SRC/js/utils/CleanJSONUtil";
 import { isSDKService } from "#PLUGINS/services/src/js/utils/ServiceUtil";
+import { findNestedPropertyInObject } from "#SRC/js/utils/Util";
 
 import {
   ROUTE_ACCESS_PREFIX,
   FRAMEWORK_ID_VALID_CHARACTERS
 } from "../constants/FrameworkConstants";
 import FrameworkUtil from "../utils/FrameworkUtil";
+import ServiceStatus from "../constants/ServiceStatus";
 
 import Application from "./Application";
 import FrameworkSpec from "./FrameworkSpec";
 
+const frameworkStatusHelper = (displayName, key, priority) => ({
+  displayName,
+  key,
+  priority
+});
+
+// prettier-ignore
+/* eslint-disable no-multi-spaces */
+const FrameworkStatus = {
+  418: frameworkStatusHelper("Initializing",                    ServiceStatus.DEPLOYING,  1),
```

Comment:
should all of these `displayName` arguments be wrapped in `i18nMark` so we can `Trans` them when they are rendered in the UI?